### ISSUE TITLE
Unable to update themes/packages

### ISSUE THREAD

**Issue by prateekkathal** [atom/atom#14526]
### Description

Unable to look for packages/theme updates

![screen shot 2017-05-20 at 3 40 32 pm](https://cloud.githubusercontent.com/assets/7609893/26274931/36c94f9e-3d73-11e7-8cc9-c2a4b28d7ec9.png)

### Steps to Reproduce

1. Open Atom
2. Go to Settings
3. Click On Updates

**Expected behavior:** List Of Updates available

**Actual behavior:** Unable to get updates

**Reproduces how often:** Everytime...

### Versions

1.17.0

### Additional Information

Running on MACOS Sierra - 10.12.4

**Comment by rlex** [comment]
as a temporary solution you can use "apm upgrade" in terminal.

**Comment by rsese** [comment]
Do you have seti-syntax or seti-ui installed @prateekkathal?

**Comment by prateekkathal** [comment]
@rlex I tried, and even that gives me an output of undefined

![screen shot 2017-05-23 at 12 59 09 am](https://cloud.githubusercontent.com/assets/7609893/26325070/0eee8124-3f53-11e7-9361-0977ecc67d15.png)

@rsese Yup, I did... And I removed those themes.... Nothing is helping.... 

Following are my current themes and packages.... 

### Themes
![screen shot 2017-05-23 at 1 01 48 am](https://cloud.githubusercontent.com/assets/7609893/26325195/92bfacf8-3f53-11e7-8fe0-6f2127214542.png)

### Packages
![screen shot 2017-05-23 at 1 02 36 am](https://cloud.githubusercontent.com/assets/7609893/26325218/a50ee1ee-3f53-11e7-8410-c0db6699b4f4.png)
![screen shot 2017-05-23 at 1 02 13 am](https://cloud.githubusercontent.com/assets/7609893/26325217/a50e220e-3f53-11e7-8b76-7d5425ac71b2.png)

**Comment by prateekkathal** [comment]
@rlex @rsese 

So I found the main issue here... 

1. I tried removing it and then downloading the latest version again. 
2. Even after that it wasn't able to fetch the updates
3. I went to `~/.atom/packages` and saw that a folder with the name `seti-syntax` was already there.... I removed it and now it works 👍 

![screen shot 2017-05-23 at 11 01 53 am](https://cloud.githubusercontent.com/assets/7609893/26339715/916bfdbc-3fa7-11e7-98d2-ac09d4e33099.png)

Sadly, I will have to re-install my old packages 😞 , I wish we had a cloud thingy with which can automatically sync our plugins/themes.
